balena.yml

```diff
@@ -1,6 +1,11 @@
 name: "Minecraft Server"
-description: "Build a Minecraft Server using a Raspberry Pi 4! Supports common Servers, SCP, RCON and Wifi Connect."
 type: "sw.application"
+description: "Build a Minecraft Server using a Raspberry Pi 4! Supports common Servers, SCP, RCON and Wifi Connect."
+fleetcta: Lets break some blocks!
+post-provisioning: >-
+  ## Usage instructions
+  Once your device joins the fleet you'll need to allow some time for it to download the application. When it's done it tries to download the server software and run it. You should then be able to connect to the server via connecting `balenaminecraftserver` locally inside minecraft.
```

Comment:
@AlexProgrammerDE is it `balenaminecraftserver` or `balenaminecraftserver.local` ?

Reply:
I normally connect through `balenaminecraftserver`. Is `.local` a mac thing?

Reply:
oh yeah could be! I think that's what I have to use but can't say for sure at the moment